[2024-08-03T11:22:08.429+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-03T11:22:08.442+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: list_postgres_s3.copy_table_dim_aisles manual__2024-08-03T04:21:56.037992+00:00 [queued]>
[2024-08-03T11:22:08.448+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: list_postgres_s3.copy_table_dim_aisles manual__2024-08-03T04:21:56.037992+00:00 [queued]>
[2024-08-03T11:22:08.448+0700] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-08-03T11:22:08.473+0700] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): copy_table_dim_aisles> on 2024-08-03 04:21:56.037992+00:00
[2024-08-03T11:22:08.475+0700] {standard_task_runner.py:64} INFO - Started process 29574 to run task
[2024-08-03T11:22:08.478+0700] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'list_postgres_s3', 'copy_table_dim_aisles', 'manual__2024-08-03T04:21:56.037992+00:00', '--job-id', '210', '--raw', '--subdir', 'DAGS_FOLDER/list_postgres_s3.py', '--cfg-path', '/tmp/tmphon5o52d']
[2024-08-03T11:22:08.479+0700] {standard_task_runner.py:91} INFO - Job 210: Subtask copy_table_dim_aisles
[2024-08-03T11:22:08.514+0700] {task_command.py:426} INFO - Running <TaskInstance: list_postgres_s3.copy_table_dim_aisles manual__2024-08-03T04:21:56.037992+00:00 [running]> on host DESKTOP-RHVHV7A.
[2024-08-03T11:22:08.583+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='list_postgres_s3' AIRFLOW_CTX_TASK_ID='copy_table_dim_aisles' AIRFLOW_CTX_EXECUTION_DATE='2024-08-03T04:21:56.037992+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-03T04:21:56.037992+00:00'
[2024-08-03T11:22:08.584+0700] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-03T11:22:08.594+0700] {sql.py:282} INFO - Executing: 
        COPY dim_aisles TO 'F:/AllCode/Data Engineer/dim_aisles.csv' CSV HEADER;
        
[2024-08-03T11:22:08.598+0700] {base.py:84} INFO - Using connection ID 'postgres_supermarket' for task execution.
[2024-08-03T11:22:08.614+0700] {base.py:84} INFO - Using connection ID 'postgres_supermarket' for task execution.
[2024-08-03T11:22:08.649+0700] {sql.py:495} INFO - Running statement: 
        COPY dim_aisles TO 'F:/AllCode/Data Engineer/dim_aisles.csv' CSV HEADER;
        , parameters: None
[2024-08-03T11:22:08.651+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-03T11:22:08.651+0700] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/common/sql/operators/sql.py", line 288, in execute
    output = hook.run(
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/common/sql/hooks/sql.py", line 442, in run
    self._run_command(cur, sql_statement, parameters)
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/common/sql/hooks/sql.py", line 500, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedTable: relation "dim_aisles" does not exist

[2024-08-03T11:22:08.655+0700] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=list_postgres_s3, task_id=copy_table_dim_aisles, run_id=manual__2024-08-03T04:21:56.037992+00:00, execution_date=20240803T042156, start_date=20240803T042208, end_date=20240803T042208
[2024-08-03T11:22:08.675+0700] {standard_task_runner.py:110} ERROR - Failed to execute job 210 for task copy_table_dim_aisles (relation "dim_aisles" does not exist
; 29574)
[2024-08-03T11:22:08.691+0700] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-08-03T11:22:08.700+0700] {logging_mixin.py:188} WARNING - /usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py:1297 AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `airflow.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
[2024-08-03T11:22:08.708+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-03T11:22:08.730+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
